web/src/portico/redirect-to-post.ts

```diff
@@ -0,0 +1,5 @@
+import $ from "jquery";
+
+$(() => {
+    $("#register, #redirect-to-post-form").trigger("submit");
```

Comment:
I think maybe if you do `.redirect-to-post-form` as a class, you can just add that class in `confirm_preregistrationuser.html`?